src/plot_api/plot_config.js

```diff
@@ -83,8 +84,10 @@ module.exports = {
     setBackground: defaultSetBackground,
 
     // URL to topojson files used in geo charts
-    topojsonURL: 'https://cdn.plot.ly/'
+    topojsonURL: 'https://cdn.plot.ly/',
 
+    // Turn all console logging on or off (errors will be thrown)
+    logging: false
```

Comment:
Oh wait. That won't work nor does it need to. More like:

```js
Plotly.setPlotConfig({ logging: 2 });
Plotly.plot(gd, data, layout);

// and then
Plotly.setPlotConfig({ logging: 0});
Plotly.plot(gd, data, layout);
```

works as expected.